fiftyone/server/mutation.py

```diff
@@ -178,31 +179,19 @@ async def set_view(
         state.selected_labels = []
 
         result_view = None
-        ds = fod.load_dataset(dataset_name)
-        view_name = None
         if saved_view_slug is not None:
             try:
+                # Load a DatasetView using a slug
+                ds = fod.load_dataset(dataset_name)
                 doc = ds._get_saved_view_doc(saved_view_slug, slug=True)
-                view_name = doc.name
+                result_view = ds._load_saved_view_from_doc(doc)
             except:
-                pass
-
-        if view_name is not None:
-            # Load a saved view by name
-
-            if ds.has_saved_view(view_name):
-                # Load a saved dataset view by name
-                result_view = ds.load_saved_view(view_name)
-                view = result_view._serialize()  # serialized view stages
-                # Set view state
-                state.view = result_view
-                state.view_name = result_view.name
-                state.saved_view_slug = saved_view_slug
-        else:
-            state.view_name = None
-            state.saved_view_slug = None
+                logging.error(
```

Comment:
I'm not sure I see the benefit to this logging. If anything, this should be an [expected error from GraphQL](https://strawberry.rocks/docs/guides/errors#expected-errors).

Reply:
oh I meant for this to be logging.debug so that it can be enabled for developers and may be helpful. I didn't want to trigger an actual error (just wanted to log it in the console) that would surface to the end user.

I'll remove it though